packages/editor/src/components/widgets/table/row/MessageRow.tsx

```diff
@@ -1,24 +1,26 @@
 import './MessageRow.css';
-import type { ComponentProps} from 'react';
+import type { ComponentProps } from 'react';
 import { forwardRef } from 'react';
 import type { MessageTextProps } from '../../message/Message';
 import { MessageText } from '../../message/Message';
 
-export type MessageRowProps = MessageTextProps & ComponentProps<'tr'>;
+export type MessageRowProps = MessageTextProps & ComponentProps<'tr'> & { colSpan: number };
 
-export const MessageRow = forwardRef<HTMLTableRowElement, MessageRowProps>(({ message, children, className, ...props }, forwardRef) => {
-  return (
-    <>
-      <tr ref={forwardRef} className={`row ${message ? `row-${message.severity.toLocaleLowerCase()}` : ''} ${className}`} {...props}>
-        {children}
-      </tr>
-      {message && (
-        <tr className='row row-message'>
-          <td colSpan={100}>
```

Comment:
Yes, I have to set it now because otherwise, the validation message completely disrupts the entire design 😅. However, I have made the field optional now, and if it is not set, I have set the colSpan to 2.